plugins/techdocs/package.json

```diff
@@ -38,12 +38,14 @@
     "@backstage/errors": "^0.1.1",
     "@backstage/integration": "^0.5.8",
     "@backstage/integration-react": "^0.1.4",
+    "@backstage/plugin-catalog": "^0.6.6",
     "@backstage/plugin-catalog-react": "^0.2.6",
```

Comment:
Just wondering if this dependency is needed, since `techdocs` is already using `catalog-react` package, which is supposed to be the package which other plugins should consume. http://backstage.io/docs/architecture-decisions/adrs-adr011#decision

But it's possible that Favorites component doesn't fit in `catalog-react`. 🤔

Reply:
I agree it is a little awkward to have plugins depend on each other, the same occurs in the api-docs plugin since that reuses `CatalogTable`:
https://github.com/backstage/backstage/blob/81a6c95b0ab08df35da72b3e518630cdcabafdcc/plugins/api-docs/package.json#L36

Based off the ADR, is it a consensus for moving components that end up being reused between multiple plugins to `x-react` packages? or should consideration be on a component by component basis?

Reply:
The following components are also imported from `@backstage/plugin-catalog`:
`EntityListContainer`, `FilterContainer`, `FilteredEntityLayout`
and the `api-docs` plugin also imports these from the `catalog` plugin.

It might make sense to move these components to `catalog-react` and refactor affected packages in a subsequent PR?